management_interface/management_interface/forms.py

```diff
@@ -19,9 +25,46 @@ class Meta:
 
 
 class CareRecipientForm(forms.ModelForm):
+    given_name = forms.CharField(max_length=64, required=True, label="Given Name")
+    family_name = forms.CharField(max_length=64, required=True, label="Family Name")
+    nhs_number = forms.CharField(max_length=16, required=True, label="NHS Number", help_text="e.g. 5991993823")
```

Comment:
```suggestion
    nhs_number = forms.CharField(max_length=12, required=True, label="NHS Number", help_text="e.g. 999 999 9999")
```

This should have max length of 12 to allow for entry without or with spaces. Modified help text since NHS Numbers should always be presented to the user in 3,3,4 format